rsts/howto/notifications.rst

```diff
@@ -121,6 +114,7 @@ Example config
 ==============
 
 .. rli:: https://raw.githubusercontent.com/flyteorg/flyteadmin/master/flyteadmin_config.yaml
```

Comment:
can we point to the config in sandbox/eks cluster?

Reply:
it doesn't have a great example not notification formatting